metadata/.gitignore

```diff
@@ -0,0 +1,14 @@
+# These are added during tree generation only.
+/cache
+/dtd
+/glsa
+/md5-cache
+/news
+/herds.xml
+/timestamp
+/timestamp.chk
+/timestamp.x
+/pkg_desc_index
+/projects.xml
+/xml-schema
+/.checksum-test-marker
```

Comment:
No, if you do `egencache` on the overlay there would be at least `md5-cache`. Some people (including me) do this for speeding up Portage operations especially when a lot of overlays are activated.